src/lib/ares_cancel.c

```diff
@@ -64,20 +63,22 @@ void ares_cancel(ares_channel channel)
       /* Cache next since this node is being deleted */
       next = ares__llist_node_next(node);
 
-      query = ares__llist_node_claim(node);
+      query                   = ares__llist_node_claim(node);
```

Comment:
and here, I suppose it wants to ... maintain the alignment from above? I find it a peculiar way to format code.

Reply:
actually, its aligning with the line below it.  But not with the line above it because I have it set not to span across lines.